Podcast/app/src/main/java/br/ufpe/cin/if710/podcast/ui/MainActivity.java

```diff
@@ -117,6 +117,35 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             });
             /**/
         }
+
+        private boolean isListDownloaded() {
+            return false;
+        }
+
+        private void saveXML(List<ItemFeed> xml) {
+            SQLiteDatabase db = PodcastDBHelper.getInstance(getApplicationContext()).getWritableDatabase();;
+            for (ItemFeed item : xml) {
+                ContentValues values = item.contentValueRepresentation();
+                long rowId = db.insert(PodcastDBHelper.DATABASE_TABLE, null, values);
+            }
+        }
+
+        private List<ItemFeed> retrieveXML() {
```

Comment:
This functionality was not implemented. Currently, the episodes are always downloaded. See [here](https://github.com/JoaoGFarias/exercicio-podcast/pull/3/files#diff-e09b5970f39b2198b2758a103359cfdbR144)